vmdb/lib/token_manager.rb

```diff
@@ -36,94 +36,42 @@ def self.configure(_namespace, options = {})
   end
 
   def self.gen_token(namespace, token_options = {})
-    token_cleanup
-    token = SecureRandom.hex(16)
-
     ts = global_token_store(namespace)
-    ts[token] = {:expires_on => Time.now.tv_sec + @config[:token_ttl]}
-    ts[token].merge!(prune_token_options(namespace, token, token_options))
+    token = SecureRandom.hex(16)
+    token_data = {:expires_on => Time.now.tv_sec + @config[:token_ttl]}
```

Comment:
serialization worked fine.  extracted token info on a second process and got my time object as expected.  I'll update and re-push.  Thanks.